data/human/free worlds intro.txt

```diff
@@ -457,1312 +461,1509 @@ mission "FW Recon 3B"
 				"Gunboat"
 	
 	on visit
+		set "fw recon: obvious gunboat"
 		dialog `A minute after you land, a Republic Gunboat, the <npc>, lands directly next to you. JJ does not approach you. You notice from your ship's logs that this Gunboat has been following you ever since you left <origin>; before you can hand over the information, you will need to escape its pursuit.`
-	
 	on complete
-		"assisted free worlds" ++
-		dialog `JJ meets you as soon as you land, and you pass off your information and photos to him. He pays you <payment> and says, "I'll be sure to let the Free Worlds Council know how helpful you have been."`
 		payment 80000
+		"assisted free worlds" ++
 		log "Gave the Free Worlds some photos and information on the Navy base on New Wales. Jean-Jacques, the militia contact, was very grateful for the help."
+		conversation
```

Comment:
Just an observation: there seems to be a lot of empty lines in this section - and in fact in a few later sections that appear to be rearranged older content.

Reply:
It may just be the diff looking weird because of how I moved some of the missions around. As suggested in the PR description, it may be better to look at this commit by commit, or you could just view the file directly instead of in the files changed tab.

Reply:
Have checked the file itself. The diff actually looks pretty reliable, there's just lots of empty lines between labels, choices, and goto's that I wouldn't expect.

Reply:
Oh. That's one of those style choices that we're inconsistent on. I like to have empty lines between parts of conversations to break things up, particularly when branches are involved as is the case with this conversation. The conversation works fine without them, I just often find it easier to work with them. Makes it easier for me to pick things out when I'm going back through to check that everything's good.